pkg/controller/webhookbootstrap/controller.go

```diff
@@ -328,6 +328,13 @@ func (c *controller) certificateRequiresIssuance(ctx context.Context, log logr.L
 		return true
 	}
 
+	// validate the uri sans are correct
+	expectedURINames := crt.Spec.URISANs
+	if !util.EqualUnsorted(pki.URLsToString(cert.URIs), expectedURINames) {
+		log.Info("certificate uri sans are not as expected, re-issuing")
+		return true
+	}
```

Comment:
🤷‍♂ we don't actually set URI SANs as part of the webhookbootstrap controller, so this isn't strictly required...